ExampleMod/Common/Systems/SystemUI.cs

```diff
@@ -0,0 +1,56 @@
+﻿using ExampleMod.Common.UI;
+using Microsoft.Xna.Framework;
+using System.Collections.Generic;
+using Terraria;
+using Terraria.ModLoader;
+using Terraria.UI;
+
+namespace ExampleMod.Common.Systems
+{
+	public class SystemUI: ModSystem
```

Comment:
missing space before `:`
```suggestion
	public class SystemUI : ModSystem
```